airflow/www/templates/airflow/dags.html

```diff
@@ -421,10 +425,6 @@ <h2>DAGs</h2>
           if (d.count > 0)
               return"cursor:pointer;"
         })
-        .on('click', function(d, i) {
-          if (d.count > 0)
```

Comment:
This condition no longer persists (all circles are now clickable). Was that intentional? I don't see a strong case for why they needed to be conditional to begin with.

Reply:
I'm also not sure why we had this conditional. I'd be happy to restore it if you think it's useful.

Reply:
No, I think we're good to omit.